styles/styles.css

```diff
@@ -379,6 +379,11 @@ main .section {
   overflow: hidden;
 }
 
+main .section.bigger-padding{
```

Comment:
does that need content changes? can it be done without?

Reply:
just the metadata, not sure how else we can indicate that this particular sections have a dynamic padding?